README.md

```diff
@@ -340,5 +347,36 @@ Applies todo changes, either adding or removing, based on batches from `getTodoB
 | add | Batch of todos to add. |
 | remove | Batch of todos to remove. |
 
+<a name="getTodoConfig"></a>
+
+## getTodoConfig(baseDir, todoConfig) ⇒
+Gets the todo configuration.
+Config values can be present in
+
+1. The package.json
+
+**Kind**: global function  
+**Returns**: - The todo config object.  
+
+| Param | Description |
+| --- | --- |
+| baseDir | The base directory that contains the project's package.json. |
+| todoConfig | The optional todo configuration. |
+
+**Example**  
+```
+{
+  "lintTodo": {
+    "daysToDecay": {
+      "warn": 5,
+			 "error": 10
+		 }
```

Comment:
Seems wrong, I think there is an extra `}` and the `"error"` line is indented differently than the others

Reply:
Ya the auto generator borked the indentation.

Reply:
Also it was actually _missing_ a `}`

Reply:
Fixed.